config/routes.rb

```diff
@@ -5,7 +5,7 @@
 
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/216605338-b4b1a7c6-5a28-4841-bd2d-6ed179212864.png)

- [ ] Your MIT link is broken :fearful: because you don't have a LICENSE file in your repository :thinking: It would be nice if you add a LICENSE file in your repository so your MIT link in your readme would work :+1:

**Note: You can add a license file using these [guidelines](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) :+1: which would be a professional readme :blush:**